Realm/Realm.SourceGenerator/ClassCodeBuilder.cs

```diff
@@ -43,16 +43,29 @@ internal class ClassCodeBuilder
         };
 
         private readonly ClassInfo _classInfo;
+        private readonly Lazy<string> _ignoreFieldAttribute;
 
         private readonly string _helperClassName;
         private readonly string _accessorInterfaceName;
         private readonly string _managedAccessorClassName;
         private readonly string _unmanagedAccessorClassName;
 
-        public ClassCodeBuilder(ClassInfo classInfo)
+        public ClassCodeBuilder(ClassInfo classInfo, GeneratorConfig generatorConfig)
         {
             _classInfo = classInfo;
 
+            _ignoreFieldAttribute = new(() =>
+            {
+                var result = "[IgnoreDataMember, XmlIgnore]";
+                var customAttribute = generatorConfig.CustomIgnoreAttribute;
+                if (!string.IsNullOrEmpty(customAttribute))
+                {
+                    result += customAttribute;
```

Comment:
Maybe we should put a newline between the two?

Reply:
I don't have strong opinion about it - both are valid C# and was thinking putting everything related on one line would make it easier to ignore if you're just reading the code.

Reply:
Yeah, maybe it makes sense. I'm also not strongly opinionated on this.